crypto/pkcs11key/key_test.go

```diff
@@ -1,4 +1,4 @@
-// +build !nopkcs11
+// +build pkcs11
```

Comment:
It's nice to have the nopkcs11 build, as cgo wants to link libtool at runtime, and it's sometimes impractical to package just the parts you need. I have a problem, for example, where I use cfssl libraries in another project and the transitive dependency graph brings in the miekg/pkcs11 package, which then requires the libltdl-dev headers to build and the binary libltdl to run.

Upshot: as long as we're importing the miekg/pkcs11 package, I'm in favor of the build tags.